src/main/app/pay/payment.ts

```diff
@@ -10,16 +10,16 @@ class Link {
 
 class Links {
   @Type(() => Link)
-  readonly self: Link
+  readonly self?: Link
```

Comment:
Lets not weaken types for mock implementation.

Reply:
I'm not, it wasn't present

Reply:
OK, I thought self link is present in retrieve response as well. Happy with this 👍